build.ps1

```diff
@@ -61,7 +63,10 @@ If (CompareVersions -actualVersion $vboxVersion -expectedVersion $virtualBoxMinV
 
 }
 
-$packerVersion = cmd.exe /c $packer -v
+    $packerVersion = cmd.exe /c "$packer" -v
+    $packerVersion = $packerVersion.split(" ")[1]

```

Comment:
`cmd.exe /c "$packer" -v` returns the version number only such as `1.4.4` this results in `$packerVersion` getting set to null and causing and error when attempting to compare the version.

Please revert this hunk as it also should not shift the indention.